kong.conf.default

```diff
@@ -87,6 +87,18 @@
                                  # itself on `proxy_listen`, and all SSL
                                  # settings will be ignored.
 
+#ssl_cipher_suite = modern       # Defines the TLS ciphers served by Nginx.
+                                 # Accepted values are 'modern', 'intermediate',
+                                 # 'old', or 'custom'.
+# Note: See https://wiki.mozilla.org/Security/Server_Side_TLS for detailed
+# descriptions of each cipher suite.
+
+#ssl_ciphers =                   # Define a custom list of TLS ciphers to be
```

Comment:
`Defines`, as per the convention followed in this configuration file (and the `ssl_cipher_suite` property).